ionic/www/js/httpHandler.factory.js

```diff
@@ -15,20 +15,38 @@ angular.module('radar')
 		  });
 	}
 
-	// httpObject.getRatings = function(callback) {
-	// 	$http.get('/api/ratings')
-	// 		.success(function(data, status) {
-	// 			var temp = data.map(function(element) {
-	// 				return {eventID: element.event_id, comment: element.comment, stars: element.stars};
-	// 			})
-	// 			// console.log("TEMP", temp);
-	// 			// console.log("CALLBACK", callback);
-	// 			callback(temp);
-	// 		})
-	// 		.error(function(data, status) {
-	// 			console.log("ERROR FOR API RATINGS");
-	// 		});
-	// }
+	httpObject.saveNewEvent = function(title, info, streetAddress1, streetAddress2, city, state, zipCode, startDateTime, endDateTime, category, coords, userId) {
```

Comment:
refactor to pass in data object directly instead of building it in request using a dozen params